website/client/readme.md

```diff
@@ -1,20 +0,0 @@
-#Running

```

Comment:
I must have deleted it by accident. How should I go about re-adding it to my branch?